x-pack/plugins/infra/common/http_api/log_alerts/chart_preview_data.ts

```diff
@@ -58,6 +59,9 @@ export type GetLogAlertsChartPreviewDataSuccessResponsePayload = rt.TypeOf<
 export const getLogAlertsChartPreviewDataAlertParamsSubsetRT: any = rt.intersection([
   rt.type({
     criteria: countCriteriaRT,
+    count: rt.type({
+      comparator: ThresholdRT.props.comparator,
```

Comment:
Good question, there was no reason for this.

Reply:
After revisiting this I think this makes sense. The `count` is mimicking the normal (base alert params) `count`, by keeping to the threshold type it keeps in sync with that, rather than the type the threshold itself uses.